### PR TITLE
do not call (ros::spin) automatically in welcome-to-jsk.l

### PR DESCRIPTION
**Pull request by 708yamaguchi**
modify program in order to add welcome-to-jsk program to fetch apps.
Please merge this with if demo on 5/26(Sat) succeed.

### CODE REVIEW

File: welcome_to_jsk_fetch/welcome-to-jsk.l [hunk 1]
```diff
@@ -122,17 +122,15 @@
...
   (ros::ros-info "[Get ready to subscribe: /go_to_point]")
 
   (ros::subscribe "go_to_point" std_msgs::string #'go-to-point)
   (ros::spin)
   )
 
-
-(main)
+(defun init()
+  (load "package://fetcheus/fetch-interface.l")
```
Comment (k-okada) [inline welcome_to_jsk_fetch/welcome-to-jsk.l]: why you load fetch-interface here. you already load at  https://github.com/708yamaguchi/jsk_demos/blob/1293265accd0d834f361b9901728d22211dea9ff/welcome_to_jsk_fetch/welcome-to-jsk.l#L3

Comment (k-okada) [reply welcome_to_jsk_fetch/welcome-to-jsk.l]: did you add undock action?

Comment (708yamaguchi) [reply welcome_to_jsk_fetch/welcome-to-jsk.l]: At my latest commit, I remove unnecessary `(load "package://fetcheus/fetch-interface.l")`.
Also, undock fetch at the start of demo, if fetch is docking.

**Comment by 708yamaguchi** [comment]
I tested my latest commit using real fetch from my smartphone, and fetch worked correctly.

**Comment by k-okada** [comment]
@708yamaguchi do we still need this?

**Comment by 708yamaguchi** [comment]
No, I close this PR.

**Pull request closed without merging**